src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingFailureSpec.cs

```diff
@@ -15,46 +15,64 @@
 using Akka.Persistence.Journal;
 using Akka.Remote.TestKit;
 using Akka.Remote.Transport;
+using Xunit;
+using Akka.Event;
+using FluentAssertions;
 
 namespace Akka.Cluster.Sharding.Tests
 {
     public class ClusterShardingFailureSpecConfig : MultiNodeConfig
     {
+        public RoleName Controller { get; private set; }
+
+        public RoleName First { get; private set; }
+
+        public RoleName Second { get; private set; }
+
         public ClusterShardingFailureSpecConfig()
         {
-            var controller = Role("controller");
-            var first = Role("first");
-            var second = Role("second");
-
-            CommonConfig = ConfigurationFactory.ParseString(@"
-                akka.loglevel = INFO
-                akka.actor.provider = ""Akka.Cluster.ClusterActorRefProvider""
-                akka.remote.log-remote-lifecycle-events = off
-                akka.cluster.auto-down-unreachable-after = 0s
-                akka.cluster.roles = [""backend""]
-                akka.persistence.journal.plugin = ""akka.persistence.journal.in-mem""
-                akka.persistence.journal.in-mem {
-                  timeout = 5s
-                  store {
-                    native = off
-                    dir = ""target/journal-ClusterShardingFailureSpec""
-                  }
-                }
-                akka.persistence.snapshot-store.plugin = ""akka.persistence.snapshot-store.local""
-                akka.persistence.snapshot-store.local.dir = ""target/snapshots-ClusterShardingFailureSpec""
-                akka.cluster.sharding.coordinator-failure-backoff = 3s
-                akka.cluster.sharding.shard-failure-backoff = 3s
-            ");
+            Controller = Role("controller");
+            First = Role("first");
+            Second = Role("second");
+
+            CommonConfig = DebugConfig(false)
+                .WithFallback(ConfigurationFactory.ParseString(@"
+                    akka.actor {
+                        serializers {
+                            hyperion = ""Akka.Serialization.HyperionSerializer, Akka.Serialization.Hyperion""
+                        }
+                        serialization-bindings {
+                            ""System.Object"" = hyperion
+                        }
+                    }
+
+                    akka.cluster.auto-down-unreachable-after = 0s
+                    akka.cluster.roles = [""backend""]
+                    akka.cluster.sharding {
+                        coordinator-failure-backoff = 3s
+                        shard-failure-backoff = 3s
+                    }
+                    akka.persistence.snapshot-store.plugin = ""akka.persistence.snapshot-store.inmem""
+                    akka.persistence.journal.plugin = ""akka.persistence.journal.memory-journal-shared""
```

Comment:
Does this really work? MNTK tests run in separate processes: shared in-memory journal won't help here, as those nodes don't share memory.